src/Kovan-DssSpell.t.sol

```diff
@@ -556,6 +600,20 @@ contract DssSpellTest is DSTest, DSMath {
 
             address[] memory slate = new address[](1);
 
+            if (chief.live() == 0) {
```

Comment:
This is not necessary, the new `chief` is already `live`, so this whole block shouldn't be added.